spec/create_release_backmerge_pull_request_spec.rb

```diff
@@ -39,7 +39,7 @@ def stub_expected_pull_requests(expected_backmerge_branches:, source_branch:, la
 
       next if branch_exists_on_remote
 
-      allow(Fastlane::Helper::GitHelper).to receive(:point_to_same_commit?).with(target_branch, source_branch).and_return(point_to_same_commit)
+      allow(Fastlane::Helper::GitHelper).to receive(:point_to_same_commit?).with("origin/#{target_branch}", "origin/#{source_branch}").and_return(point_to_same_commit)
```

Comment:
Unrelated to this particular line, but I wonder if it would be worth adding a test to ensure that if we pass branches that already have "origin/" in the name, our logic doesn't append an additional "origin/" to them.